pages/your_tasks/writing_readable_code.md

```diff
@@ -79,13 +79,15 @@ Here is the summary of some of the practices to follow to make your code more re
 - Use descriptive names for function, class and variable names that help explain their purpose.
 - Keep lines short - avoid long lines, and instead write blocks of lines that are horizontally short and vertically long. 
 - Use indentation to show the hierarchy of your code and mark the beginning and end of control structures. 
+- If possible use type-annotations for untyped languages like, for example, python or java-script. In non-trivially sized software projects type annotations are a great help to understand code. Use a type-checker to validate your code, for example, {% tool "mypy" %} for python. This will reduce the risk of faulty type-annotations (it will also improve the quality of your code by detecting faulty assumptions)
```

Comment:
```suggestion
- If possible use type annotations for untyped languages like Python or JavaScript. Type annotations (also known as type hints) are a way to specify the *expected* data types of variables, function parameters, and return values in progamming languyages that are not strongly typed and are of great help to understand code. Use a type checker to validate your code, for example, {% tool "mypy" %} for Python. This will reduce the risk of faulty type annotations (it will also improve the quality of your code by detecting faulty assumptions)
```